site/app.yaml

```diff
@@ -10,3 +10,19 @@ handlers:
 - url: /(.*)
   static_files: _site/\1
   upload: _site/(.*)
+
+skip_files:
+- ^_apps/.*
+- ^_contributors/.*
+- ^_includes/.*
+- ^_layouts/.*
+- ^_sass/.*
+- ^assets/.*
+- ^node_modules/.*
+- ^_config\.yml$
+- ^Gemfile.*
+- ^package\.json$
+- ^sw-precache-config\.js$
+- ^(.*/)?.*\.md$
+- ^(.*/)?.*\.gpg$
+- ^(.*/)?\..*$
```

Comment:
From the docs, it mentions that the default skipped files config is:

```
skip_files:
- ^(.*/)?#.*#$
- ^(.*/)?.*~$
- ^(.*/)?.*\.py[co]$
- ^(.*/)?.*/RCS/.*$
- ^(.*/)?\..*$
```

Do you think we should include all of the defaults along with all the other skipped files you included since we're _extending_ it? Just wondering so please correct me if I'm wrong.